### PR TITLE
Retrieve denom from config or the chain registry

### PR DESCRIPTION
**Pull request by andynog**
closes: #9 

Implemented logic to retrieve the denom from the configuration file, if it is not in the config file then try to pull it from the chain registry (https://github.com/cosmos/chain-registry). The denom is currently used in the `tx vote` and `tx withdraw`

> NOTE: The information from the chain registry could be leveraged one day for other features which is nice.

### CODE REVIEW

File: main.go
Comment (greg-szabo) [inline main.go:73]: Shouldn't we be able to override this from the command-line too?

We don't have many command-line parameters and I'm not sure what the best way to pursue: only use the config file for parameters or allow the command-line to override things too?

**Review by greg-szabo — COMMENTED** [review]
What's the priority order on parameter override?

I assume this:
`command-line parameter > config file > chain registry`

But, I'm a bit confused if the chain registry or the config file should have higher priority.

File: main.go
Comment (andynog) [reply main.go:73]: good point, I think it makes sense to have an option to override. One use case for that I believe it will be possible to pay fees with another denom other than native. So overriding would make sense in this case too.

**Comment by andynog** [comment]
Please don't merge it yet. Forgot to add some information to the README about the denom in the configuration

**Comment by andynog** [comment]
@greg-szabo added a few more changes and I think it's good to merge if no more feedback. Thanks!

**Review by greg-szabo — APPROVED** [review]

**Pull request merged**
